tests/e2e/performance/test_fio_benchmark.py

```diff
@@ -22,6 +22,20 @@
 log = logging.getLogger(__name__)
 
 
+@pytest.fixture(scope="function")
+def ripsaw(request):
+
+    # Create benchmark Operator (formerly ripsaw)
+    ripsaw = RipSaw()
+
+    def teardown():
+        ripsaw.cleanup()
+        time.sleep(10)
```

Comment:
does it take only 10 seconds to clean up ripsaw (operator benchmark and the project)? should we make it 30 seconds.

Reply:
no, in the ripsaw.cleanup() method there is a wait-for resource() function which wait for deletion, this is 10 sec extra.